libs/datamodel/connectors/datamodel-connector/src/lib.rs

```diff
@@ -161,6 +161,10 @@ pub trait Connector: Send + Sync {
         self.has_capability(ConnectorCapability::CompoundIds)
     }
 
+    fn supports_types(&self) -> bool {
```

Comment:
Nitpick: depending on what we land on as a name for the feature, that should become something like `supports_composite_types()`, right?

Reply:
Yes, the naming will probably change slightly throughout.